mcs/class/referencesource/mscorlib/system/throwhelper.cs

```diff
@@ -593,7 +593,8 @@ internal enum ExceptionArgument {
         byteOffset,
         minimumBufferSize,
         offset,
-        values
+        values,
+        comparisonType
```

Comment:
You could have `comparisonType,` to avoid the 2-line changes next time you need to add a value to this enum.